ga4gh/templates/landing_message.html

```diff
@@ -0,0 +1,11 @@
+<!-- //
+    If you would like a different landing message to appear you can modify
+    this file, or set the `LANDING_MESSAGE_HTML` configuration value to
+    point to an HTML snippet.
+//-->
+<div>
+    Welcome to the GA4GH reference server landing page! This page describes
+    the available operations, configuration, and datasets made available by
+    this server instance. For more information please visit
+    <a href="https://github.com/ga4gh/server">our github repository.</a>
```

Comment:
Do we want to add the default support address here (ga4gh-server-support@soe.ucsc.edu)?